### DISCUSSION TITLE
Where to download zxing_reader.js and zxing_reader.wasm except build source code

### DISCUSSION THREAD

**Discussion by 549393092** [zxing-cpp/zxing-cpp#768] [General]
Dear,

I am new to wasm, I would like to learn the demo from [demo_cam_reader.html](https://github.com/zxing-cpp/zxing-cpp/blob/master/wrappers/wasm/demo_cam_reader.html).
As [readme](https://github.com/zxing-cpp/zxing-cpp/blob/master/wrappers/wasm/README.md) stated, it has to build to create zxing.js and zxing.wasm (and _reader/_writer versions). But I encountered many troubles to build it, so is there any place that I can download them directly?

Thanks and Best Regards

**Comment by axxel** [comment]
There are 3 options:
a) look at the [wasm-artifacts.zip](https://github.com/zxing-cpp/zxing-cpp/releases/download/v2.2.0/wasm-artifacts.zip) latest release page.
b) look at the latest [ci workflow](https://github.com/zxing-cpp/zxing-cpp/actions/workflows/ci.yml), there you also find a `wasm-artifacts` zip file.
c) download it from the github pages publication: [zxing_reader.js](https://zxing-cpp.github.io/zxing-cpp/zxing_reader.js), [zxing_reader.wasm](https://zxing-cpp.github.io/zxing-cpp/zxing_reader.wasm), etc.

**Reply by 549393092** [reply]
@axxel Thanks a lot!
